app/models/user.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id              :integer          not null, primary key
+#  name            :string
+#  email           :string
+#  password_digest :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+class User < ApplicationRecord
+  PASSWORD_FORMAT = /\A(?=.*\d)(?=.*[a-z])/x # password should contain only digits and letters
+  has_secure_password
```

Comment:
why we added this line? how it works?

Reply:
This is the formatting for when we want `password should contain only digits and letters`
link https://stackoverflow.com/a/33632569/6371134